main.py

```diff
@@ -0,0 +1,82 @@
+# This is a sample Python script.
+
+# Press Shift+F10 to execute it or replace it with your code.
+# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+
+
+# Use a breakpoint in the code line below to debug your script.
+
+
+# Press the green button in the gutter to run the script.
+
+
+# See PyCharm help at https://www.jetbrains.com/help/pycharm/
+def main():
+    first_board = [[], [], [], [], [], [], [], [], [], []]
+    for g in range(0, 10):
+        for k in range(0, 10):
+            first_board[g].append(0)
+            print(first_board[g][k], end=' ')
+        print()
+    z = (input())
```

Comment:
2 things:
* storing the value of the input in `z` is redundant you don't use `z` afterwards
* remove those inputs all together I don't think they make too much sense